packages/app/cypress/e2e/studio/README.md

```diff
@@ -0,0 +1,5 @@
+## Cypress Studio Tests
+
+These are the tests for the Cypress Studio feature. [Learn more here](https://docs.cypress.io/guides/references/cypress-studio).
+
+Due to an (currently unknown) reason, if more than 1 test is written per spec file, the second test will fail. For this reason, each file has one test.
```

Comment:
what is the second test's error message?

Reply:
No error, turns out it was a race condition and I fixed it. Now all the studios test are in a single `studio.spec.ts` file. :100:

Reply:
Commit consolidating them: https://github.com/cypress-io/cypress/pull/23544/commits/ffc10d415e1445af1c8b593f2736ad68276cd7fd